lib/src/datasource/local/database.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:drift/drift.dart';
+import 'dart:io';
+import 'package:drift/native.dart';
+import 'package:path_provider/path_provider.dart';
+import 'package:path/path.dart' as p;
+import 'package:sequence/src/datasource/local/converters/apple_music_db.dart';
+import 'package:sequence/src/datasource/local/converters/spotify_db.dart';
+
+part 'database.g.dart';
+
+@DataClassName('RecognitionResultD')
+class RecognitionResultDb extends Table {
+  TextColumn get artist => text()();
```

Comment:
**question (non blocking)**: are the double parenthesis normal ?

Reply:
Yes they are and are actually required to define a column in the drift database.
Kindly refer to https://drift.simonbinder.eu/docs/getting-started/ at your convenience.